tools/eval/run_register.py

```
@@ -352,95 +660,237 @@ def main():
+        parser.error("--model is required with --runner codex")
+    if args.runner == "claude" and args.model:
+        parser.error("--model is only valid with --runner codex")
+    baseline_root = args.baseline.resolve() if args.baseline else SKILLS
+    if not baseline_root.is_dir():
+        parser.error(f"baseline tree is not a directory: {baseline_root}")
     variant_root = args.variant.resolve() if args.variant else None
     if variant_root and not variant_root.is_dir():
         parser.error(f"variant tree is not a directory: {variant_root}")
 
-    cases = json.loads(CASES.read_text())["cases"]
+    version = subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)